### DISCUSSION TITLE
The website should redirect to HTTPS

### DISCUSSION THREAD

**Discussion by jviotti** [json-schema-org/community#39] [General]
Not sure if this is intentional behavior (for backwards-compatibility purposes perhaps?), but every page of the json-schema.org website can be accessed over plain HTTP.  The website doesn't transfer sensitive information, but it is a considered a good practice to redirect HTTP to HTTPS given that HTTPS is already supported.

This can be configured purely through the GitHub Pages settings as far as I remember given my own personal website:

<img width="711" alt="Screen Shot 2021-08-20 at 22 11 46" src="https://user-images.githubusercontent.com/2192773/130307326-030acb92-f634-4e09-93fb-913d2e921b63.png">

Attempts to access the website over plain HTTP causes the expected redirection:

```
$ http http://www.jviotti.com
HTTP/1.1 301 Moved Permanently
...
Location: https://www.jviotti.com/
...
```

**Comment by Relequestual** [comment]
The checkbox for us is disabled for reasons I'd have to investigate.
I can't see any reason why we wouldn't enable this now. I thought we already had done previously...

**Reply by Relequestual** [reply]
I've re-opened https://github.com/json-schema-org/json-schema-org.github.io/issues/236
